packages/devtools_app/lib/src/shared/diagnostics/inspector_service.dart

```diff
@@ -35,14 +35,16 @@ abstract class InspectorServiceBase extends DisposableController
     required this.clientInspectorName,
     required this.serviceExtensionPrefix,
     required String inspectorLibraryUri,
+    ValueListenable<IsolateRef?>? evalIsolate,
   })  : assert(serviceConnection.serviceManager.connectedAppInitialized),
         assert(serviceConnection.serviceManager.service != null),
-        clients = {},
-        inspectorLibrary = EvalOnDartLibrary(
-          inspectorLibraryUri,
-          serviceConnection.serviceManager.service!,
-          serviceManager: serviceConnection.serviceManager,
-        ) {
+        clients = {} {
+    inspectorLibrary = EvalOnDartLibrary(
+      inspectorLibraryUri,
+      serviceConnection.serviceManager.service!,
+      serviceManager: serviceConnection.serviceManager,
+      isolate: evalIsolate,
+    );
```

Comment:
can this go back to being outside of the constructor block so that inspectorLibrary doesn't have to be late?